cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -17,9 +17,12 @@
 # https://docs.gitlab.com/ee/ci/yaml/workflow.html#switch-between-branch-pipelines-and-merge-request-pipelines
 workflow:
   rules:
-     - if: $CI_EXTERNAL_PULL_REQUEST_IID && $CI_COMMIT_BRANCH
-     - if: $CI_COMMIT_BRANCH =~ $CI_DEFAULT_BRANCH
-       when: never
+    - if: $CI_EXTERNAL_PULL_REQUEST_IID &&
+          $CI_EXTERNAL_PULL_REQUEST_SOURCE_REPOSITORY == $CI_EXTERNAL_PULL_REQUEST_TARGET_REPOSITORY &&
+          $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == $CI_DEFAULT_BRANCH
+    - if: $CI_COMMIT_BRANCH =~ $CI_DEFAULT_BRANCH
```

Comment:
what does this mean?

Reply:
The intention with these rules is that only PRs originated within `ensembl-genomio` and against `main` will trigger an automatic run of the pipeline. For commit, only those done against `main` should trigger an automatic run of the pipeline, but not for any other (they can always be triggered manually from GitLab if needed).

Reply:
Don't we have "when: never" on the following line?

Reply:
Based on my interpretation of the documentation, if no rules apply, the pipeline is not created, e.g. https://docs.gitlab.com/ee/ci/yaml/workflow.html#workflow-rules-examples